static/js/components/ObjPageAnnotations.jsx

```diff
@@ -69,11 +69,21 @@ const getMuiTheme = (theme) =>
   });
 
 // Table for displaying Obj annotations on Candidate page and Source page
-const ObjPageAnnotations = ({ annotations }) => {
+const ObjPageAnnotations = ({ annotations, spectra = [] }) => {
   const classes = useStyles();
   const theme = useTheme();
   const renderValue = (value) => getAnnotationValueString(value);
   const renderTime = (created_at) => dayjs().to(dayjs.utc(`${created_at}Z`));
+  const renderSpectrumDate = (observed_at) => {
+    if (observed_at) {
+      const dayFraction = (parseFloat(observed_at.substring(11, 13)) / 24) * 10;
+      return [observed_at.substring(0, 10), ".", dayFraction.toFixed(0)].join(
+        ""
+      );
```

Comment:
```suggestion
      return observed_at.substring(0, 10) + "." + dayFraction.toFixed(0);
```

Reply:
Simpler and easier to maintain

Reply:
or perhaps preferably:
``` javascript
return `${observed_at.substring(0, 10)}.${dayFraction.toFixed(0)}`;
```

Reply:
I will go with option 2 because option 1 is what I did but the Lint didn't like that.